### DISCUSSION TITLE
Set a level for headings in the toc.

### DISCUSSION THREAD

**Discussion by dev-universal-parallel** [fuma-nama/fumadocs#939] [Q&A] [answered]
Let me start by saying that this project is truly wonderful, thank you so much for all the hard work and dedication!
My question is if it's possible to set a level on the headings that make it to the table of contents, for instance only ### and "higher" (## and #)? 
Or is there a way to disable a heading from making it in? 
Many thanks!

**Comment by fuma-nama** [comment, marked as answer]
1. There is no built-in way to change the depth of its toc item without affecting the heading level. The only option is to manipulate the TOC array in your page.tsx passed to the DocsPage component.

3. You can use the MDX syntax, instead of markdown syntax. The heading JSX elements won't be included in TOC.

**Reply by dev-universal-parallel** [reply]
Perfect, I will try those then, thank you very much!
